ddtrace/internal/ci_visibility/recorder.py

```diff
@@ -778,6 +805,7 @@ def _on_discover_session(
     )
 
     CIVisibility.add_session(session)
+    CIVisibility.set_test_session_name(test_command=discover_args.test_command)
```

Comment:
We can add it, but we still need the `set_test_session_name()` call to trigger the metadata modification in the writer. Or do we have a better way to do it?

Maybe this will be easier to do when we move everything to the new test API.

Reply:
Yeah let's hold off.